<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserTableWindow</class>
 <widget class="QMainWindow" name="UserTableWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>812</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load User Table</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="sourcesTab">
       <attribute name="title">
        <string>Sources</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="settingsContainer" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item alignment="Qt::AlignTop">
             <widget class="QGroupBox" name="searchSettingContainer">
              <property name="title">
               <string>Search settings</string>
              </property>
              <layout class="QFormLayout" name="formLayout_2">
               <item row="3" column="0">
                <widget class="QLabel" name="selectionLabel">
                 <property name="text">
                  <string>Selection</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QComboBox" name="selectionComboBox">
                 <item>
                  <property name="text">
                   <string>Point</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Rectangular</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="radiusLabel">
                 <property name="text">
                  <string>Radius</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QLineEdit" name="radiusLineEdit">
                 <property name="text">
                  <string>0.1</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="dlLabel">
                 <property name="text">
                  <string>dl</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QLineEdit" name="dlLineEdit">
                 <property name="text">
                  <string>0.1</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="0">
                <widget class="QLabel" name="dbLabel">
                 <property name="text">
                  <string>db</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="1">
                <widget class="QLineEdit" name="dbLineEdit">
                 <property name="text">
                  <string>0.1</string>
                 </property>
                </widget>
               </item>
               <item row="7" column="1">
                <widget class="QPushButton" name="queryButton">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Query</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Designation</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="idBox"/>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Longitude</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="glonBox"/>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Latitude</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QComboBox" name="glatBox"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QTableWidget" name="sourcesTable">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="imagesTab">
       <attribute name="title">
        <string>Images</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
          <widget class="QScrollArea" name="imagesScrollArea">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="imagesScrollAreaContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>198</width>
              <height>646</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2"/>
           </widget>
          </widget>
          <widget class="QTableWidget" name="imagesTable">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
           <column>
            <property name="text">
             <string/>
            </property>
           </column>
           <column>
            <property name="text">
             <string>designation</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>glon</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>glat</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string># cutouts</string>
            </property>
           </column>
          </widget>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QPushButton" name="downloadImagesButton">
          <property name="text">
           <string>Download</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="cubesTab">
       <attribute name="title">
        <string>Cubes</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QSplitter" name="splitter_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
          <widget class="QScrollArea" name="cubesScrollArea">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>0</height>
            </size>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="cubesScrollAreaContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>178</width>
              <height>646</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4"/>
           </widget>
          </widget>
          <widget class="QTableWidget" name="cubesTable">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
           <column>
            <property name="text">
             <string/>
            </property>
           </column>
           <column>
            <property name="text">
             <string>designation</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>glon</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>glat</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string># cubes</string>
            </property>
           </column>
          </widget>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QPushButton" name="downloadCubesButton">
          <property name="text">
           <string>Download</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>idBox</tabstop>
  <tabstop>glonBox</tabstop>
  <tabstop>glatBox</tabstop>
  <tabstop>selectionComboBox</tabstop>
  <tabstop>radiusLineEdit</tabstop>
  <tabstop>dlLineEdit</tabstop>
  <tabstop>dbLineEdit</tabstop>
  <tabstop>queryButton</tabstop>
  <tabstop>sourcesTable</tabstop>
  <tabstop>imagesScrollArea</tabstop>
  <tabstop>imagesTable</tabstop>
  <tabstop>cubesScrollArea</tabstop>
  <tabstop>cubesTable</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
